config/settings.yml

```diff
@@ -32,6 +32,13 @@ discourse_solved:
     client: true
   notify_on_staff_accept_solved:
     default: false
+  prevent_assign_notification:
+    default: false
+  assign_to_status_on_solved:
```

Comment:
I actually think we don't need this site setting. You can default `status_to_assign_on_solved` to an empty string rather than `Done` and use a `present?` or `blank?` check in plugin.rb. If a string is present we basically know that this is enabled.